### PR TITLE
Add eslint and format code using standard style

### PR DESCRIPTION
**Pull request by anxolin**
Prettier plugin was messing up, so I added some basic rules for eslint so we use standard code style

### CODE REVIEW

**Comment by anxolin** [comment]
> Also, remove the package-lock and let's add yarn instructions to the README
Done in another PR later. You'll see, it's there already :)

> What's wrong with the prettier..
Nothing. Prettier is used too. Just check next PR

**Comment by W3stside** [comment]
> > Also, remove the package-lock and let's add yarn instructions to the README
> > Done in another PR later. You'll see, it's there already :)
> 
> > What's wrong with the prettier..
> > Nothing. Prettier is used too. Just check next PR

ah yeah i saw. maybe these PRs are TOO small. anyways, good stuff. merge it

**Comment by anxolin** [comment]
> these PRs are TOO small. anyways, good stuff. merge it

This one formats the code using `standard`. I don't want to mix this with other PRs. That's why.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
```
Comment (Velenir) [inline .eslintrc.js]: You need a ts parser like we do [here](https://github.com/gnosis/dex-react/blob/develop/.eslintrc.js#L2)
and maybe extend a plugin or two for TS

Comment (Velenir) [reply .eslintrc.js]: Actually, not using the parser may be the reason for prettier problems

Comment (Velenir) [reply .eslintrc.js]: Wrong PR 😓 
But add it for TS

**Pull request merged**